documents/model.go

```diff
@@ -74,6 +74,9 @@ type Model interface {
 	// GetCollaborators returns the collaborators of this Document.
 	// filter ids should not be returned
 	GetCollaborators(filterIDs ...identity.CentID) ([]identity.CentID, error)
+
+	// Roles returns the Roles in this Document
+	Roles() []coredocumentpb.Role
```

Comment:
We should not expose roles. What is the use case ?

Reply:
in the mintNFTWithProofs test, we want to access the role count from the Model